apps/frontend/src/app/pages/company/gpts/gpts.component.html

```diff
@@ -0,0 +1,347 @@
+<div class="container mx-auto min-h-screen px-3 sm:px-0">
+  <div class="pb-4 pt-6 flex flex-wrap justify-between gap-6">
+    <h2 class="font-sans font-medium text-4xl text-base-100">GPTs</h2>
+    @if(gptType === 'FINETUNING'){
+    <button
+      class="btn btn-primary btn-outline w-52"
+      onclick="create_fine_turning_modal.showModal()"
+    >
+      Create Fine Tuning
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/ec7f48cb-1a95-4459-b37e-5b930ecbda46)

When I hover over one of the options the other options jump around. It looks a little weird.